README.md

```diff
@@ -62,4 +62,8 @@ If you need a fresh database:
      docker volume rm your-rents-services-geodata_postgres-yrs-geodata
      ```
 
-Then restart the service.
+- Then restart the service:
+
+     ```shell
+     docker compose up -d
```

Comment:
I meant "restart the geodata service", so the suggested command should be:

```shell
./mvnw spring-boot:run
```

...or anything you could use for starting the geodata service for development.